### PR TITLE
add TheAlgorithms icon #6912

### PR DESCRIPTION
**Pull request by jang-jaehyuk**
**Issue:** #6912 
**Alexa rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
hi @jang-jaehyuk and @service-paradis,

i checked the error and its to the line seperator. Currently u use CR, which ads a second line on linux/macos system.

Can you switch to LF (linux and macos) style, than everything is working

image

File: icons/thealgorithms.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>The Algorithms</title><path d="M8.226,19.857H1.353l1.79-4.225h4.812L13.308,3.21H7.564l-4.226,9.82h2.587c.18.3.511.51.887.51a1.04,1.04,0,0,0,1.038-1.037c0-.572-.467-1.023-1.038-1.023-.421,0-.767.24-.932.602H4.647l3.503-7.94h3.76L7.383,14.684l-4.766.03L0,20.79h8.842L10,18.263h3.835l1.278,2.526H24L15.985,3.211Zm2.27-2.586,1.384-3.023,1.503,3.023zm5.218,2.691-.872-1.759h2.737c.18.33.526.556.917.556a1.04,1.04,0,0,0,1.038-1.037,1.04,1.04,0,0,0-1.038-1.038c-.42,0-.782.256-.947.617H14.42l-2.09-4.06,1.534-3.369,1.729,3.519h.812c.165.346.526.601.932.601a1.04,1.04,0,0,0,1.038-1.037,1.04,1.04,0,0,0-1.038-1.038c-.436,0-.812.271-.962.662h-.3l-1.79-3.64,1.699-3.728,6.677,14.751Z"/></svg>
```
Comment (mondeja) [inline icons/thealgorithms.svg:1]: Hi @jang-jaehyuk

The linter is failing because the last newline character (after `</svg>`)  must be removed.

**Review by mondeja — CHANGES_REQUESTED** [review]

File: icons/thealgorithms.svg (hunk 1 above)
Comment (jang-jaehyuk) [reply icons/thealgorithms.svg]: thank you very much :)

**Review by dirien — APPROVED** [review]
Hi @jang-jaehyuk,

perfect fit! 
image

File: icons/thealgorithms.svg (hunk 1 above)
Comment (service-paradis) [reply icons/thealgorithms.svg]: Thank you @dirien and @mondeja, totally missed this one 😅 
We might try to get a better message for this case?

**Pull request merged**